CHANGELOG.md

```diff
@@ -1,3 +1,225 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns metadata syncing on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Adds `citus.max_client_connections` GUC to limit non-Citus connections
+
+* Allows locally creating objects having a dependency that cannot be distributed
+
+* Distributes aggregates with `CREATE AGGREGATE` command
+
+* Distributes functions with `CREATE FUNCTION` command
+
+* `citus_shard_indexes_on_worker` shows all local shard indexes regardless of
+  `search_path`
+
+* `citus_shards_on_worker` shows all local shards regardless of `search_path`
+
+* Adds `citus.create_object_propagation` GUC to control DDL creation behaviour
+  in transactions
+
+* Hides shards based on `application_name` prefix
+
+* Prevents specifying `application_name` via `citus.node_conninfo`
+
+* Starts identifying rebalancer backends by `application_name=citus_rebalancer`
+
+* Starts identifying internal backends by `application_name=citus_internal`
+
+* Adds `citus.enable_unsafe_triggers` flag to enable unsafe triggers on
+  distributed tables
+
+* Adds `fix_partition_shard_index_names` UDF to fix currently broken names
+
+* Adds propagation for foreign server commands
+
+* Adds propagation of `CREATE AGGREGATE` commands
```

Comment:
We already have that above as "Distributes aggregates with `CREATE AGGREGATE` command"